src/Java.Interop/Directory.Build.targets

```diff
@@ -16,14 +16,11 @@
       Inputs="$(_JNIEnvGenPath)"
       Outputs="Java.Interop/JniEnvironment.g.cs;$(IntermediateOutputPath)jni.c">
     <MakeDir Directories="$(IntermediateOutputPath)" />
-    <PropertyGroup>
-      <_AddCompile Condition=" !Exists('Java.Interop/JniEnvironment.g.cs') ">True</_AddCompile>
-    </PropertyGroup>
     <Exec
         Command="$(_RunJNIEnvGen) Java.Interop/JniEnvironment.g.cs $(IntermediateOutputPath)jni.c"
     />
     <ItemGroup>
-      <Compile Include="Java.Interop/JniEnvironment.g.cs" Condition=" '$(_AddCompile)' == 'True' " />
+      <Compile Include="$([System.IO.Path]::Combine('Java.Interop','JniEnvironment.g.cs'))" KeepDuplicates="false" />
```

Comment:
Just an idea, could this file be in `$(IntermediateOutputPath)JniEnvironment.g.cs` instead?

Then you wouldn't need to worry about it getting automatically included at all? You just need to make sure the file is added to `@(FileWrites)` where it is generated.

Reply:
Let's just go with what we got for now to get the build fixed.